web/src/app/components/LobbyPlayerList.tsx

```diff
@@ -10,7 +10,7 @@ export default function LobbyPlayerList({players} : LobbyPlayerListProps) {
             <h2>Connected Players ({Object.keys(players).length}/4)</h2>
             <ul>
                 {Object.entries(players).map(([id, {username}], index) => (
-                    <div className="mt-1">
+                    <div key={id + username} className="mt-1">
```

Comment:
Needed to add a key here, as react was always complaining about child components not having keys when rendering an array of components.